manifests/params.pp

```diff
@@ -91,6 +91,7 @@
   $conf_d                     = "${confdir}/conf.d"
   $config_file                = "${confdir}/unbound.conf"
   $control_enable             = false
+  $control_setup_path         = '/usr/sbin/unbound-control-setup'
```

Comment:
It needs to be fully specified, otherwise the catalog does not apply. If different platforms have it elsewhere, just move it in params to select based on OS/etc or override with hiera.